contentcuration/contentcuration/viewsets/assessmentitem.py

```diff
@@ -109,8 +133,7 @@ def set_files(self, all_objects, all_validated_data=None):  # noqa C901
             # have had these fields modified.
             md_fields_modified = {
                 self.id_value_lookup(ai) for ai in all_validated_data
-                    if "question" in ai or "hints" in ai or "answers" in ai
-            }
+                if "question" in ai or "hints" in ai or "answers" in ai}
```

Comment:
The changes to these lines looks like they can be reverted-- the prior formatting seems appropriate

Reply:
Done!